scripts/gulpfiles/build_tasks.js

```diff
@@ -283,8 +313,8 @@ function chunkWrapper(chunk) {
   const amdDeps = fileNames.join(', ');
   const cjsDeps = fileNames.map(f => `require(${f})`).join(', ');
   const browserDeps =
-      chunk.dependencies.map(d => `root.${d.namespace}`).join(', ');
-  const imports = chunk.dependencies.map(d => d.namespace).join(', ');
+      chunk.dependencies.map(d => `root.${d.exports}`).join(', ');
+  const imports = chunk.dependencies.map(d => d.importAs).join(', ');
```

Comment:
Every chunk previously had a `namespace` property, but not all chunks have an `importAs` property (in fact, only one does). Is that an issue?

Reply:
Only chunks which are depended upon by other chunks need an `importAs`, and at the moment only the first chunk is depended upon by any others.

Reply:
(Arguably I should add a robustness check here, such that if it discovers a chunk that has become a dependancy but has no `importAs` it will complain.  Feel free to open a bug about that if you like…)

Reply:
I see. Then this works fine here, but it's fragile.

Reply:
I'll file a low priority bug after I finish this review

Reply:
After reading more: I expect that this code will continue to change into next quarter, particularly when we mess around with entry points, so I'll hold off on that bug.